hoomd/md/pair/aniso.py

```diff
@@ -477,7 +477,8 @@ def __init__(self, nlist, default_r_cut=None, mode='none'):
                               rounding_radii=OnlyIf(
                                   to_type_converter((float, float, float)),
                                   preprocess=self._to_three_tuple),
-                              len_keys=1))
```

Comment:
Run `pre-commit` either locally or on the PR to fix the code style of these lines (see instructions at the bottom of the page here: https://results.pre-commit.ci/run/github/147663007/1664224124.eYtD4HXeQ0e2r02RvT12Sw)

Reply:
I really like that it doesn't merge fixes automatically!

Reply:
Yes, I opted in to that setting: https://github.com/glotzerlab/hoomd-blue/blob/c46d4af2268fab414e87ec58786050862035dda9/.pre-commit-config.yaml#L1-L4
I had way too many local merge conflicts after pushing and then making more commits locally.